packages/api/src/commands/install.ts

```diff
@@ -86,6 +86,10 @@ cmd
         return res;
       })
       .then(Oas.init)
+      .then(async spec => {
+        await spec.dereference({ preserveRefAsJSONSchemaTitle: true });
+        return spec;
+      })
```

Comment:
This is where we weren't running dereferencing. I noticed it because while generating the Star Trek SDK I originally ran it through the codegen test suite to give me my expected files, and began creating those files, before saying screw this and did this:

```bash
npm run debug:bin -- install https://github.com/readmeio/oas-examples/blob/main/3.0/json/star-trek.json
```

The result of that SDK was **wildly** different, and missing a ton of schema files, from what the test suite was expecting to be present.